### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by peter-mouland**
might seem weird to do this on a demo app (which i'm grateful you left alive!), but the app helped me out.  The only thing I didn't notice was the istanbul version, so I thought i'd highlight it in a readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# istanbul-no-coverage
+Demonstrate how to collect code coverage from Istanbul, Mocha and BAbel 6
```
Comment (JakeSidSmith) [inline README.md]: You've got a capital A in babel.

Comment (JakeSidSmith) [inline README.md]: Also, I don't think the "6" is necessary. :)

**Comment by JakeSidSmith** [comment]
Couple of changes and I'll merge this. :)

**Comment by peter-mouland** [comment]
great - simple things sometimes help!
Let me know if you can't squash the commits, i'll do it on the CLI (so far this has all been done with github)

**Pull request merged**